ProjectFiles/Assets/Scripts/Gameplay/ParkingSpace.cs

```diff
@@ -39,7 +38,7 @@ void OnTriggerEnter(Collider other)
         {
             if (other.TryGetComponent(out VehicleDriver driver))
             {
-                mat.color = new Color(0,1,0,0.3f);
+                mat.color = new Color(1, 1, 0, 0.3f);
                 SpaceEnterEvent?.Invoke(0, SpaceID);
             }
             else if (other.TryGetComponent(out Vehicle.Vehicle v))
```

Comment:
Nitpick, but can `Vehicle` be imported because `Vehicle.Vehicle` looks bad?

Reply:
We tried, but because Vehicle is the namespace too it gets angry and makes a lot of red :cry: